src/selectors/index.ts

```diff
@@ -19,9 +18,9 @@ export const getQuery = createSelector(
 );
 
 export const getMainResult = createSelector(
-  getQuery, getSchema, getData,
-  (query: Query, schema: Schema): SpecQueryModelGroup => {
-    return recommend(query, schema).result;
+  getRecommends,
```

Comment:
i agree. i was just keeping it out of the root `getRecommends` in case we need more values for the main panel. (also it was nice for debugging).